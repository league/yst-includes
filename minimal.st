
~~~~ {.python include="minimal.py"}
REPLACE ME
~~~~

