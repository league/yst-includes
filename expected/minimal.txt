~~~~ {.python include="minimal.py"}
print "Hello, world."
~~~~
